control-plane/tests/test_cli_inspection.py

```
@@ -589,6 +590,272 @@ def __init__(self, server_address: tuple[str, int], handler_class: type) -> None
+                    diagnostics_payload["latest_reconciliation"]["reconciliation_id"],
+                    reconciliation.reconciliation_id,
+                )
+            finally:
+                if servers:
+                    servers[0].shutdown()
+                thread.join(timeout=2)
+
+    def test_service_emits_structured_observability_logs_for_live_path_and_fail_closed_rejection(
+        self,
+    ) -> None:
+        _store, service, promoted_case, evidence_id, reviewed_at = self._build_phase19_in_scope_case()
```

Ruff (0.15.9)

[warning] Unpacked variable `reviewed_at` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)